lib/config/presets/internal/regex-managers.spec.ts

```diff
@@ -0,0 +1,222 @@
+import { regexMatches } from '../../../../test/util';
+import { extractPackageFile } from '../../../modules/manager/regex';
+import { presets } from './regex-managers';
+
+describe('config/presets/internal/regex-managers', () => {
+  describe('Update `_VERSION` variables in Dockerfiles', () => {
+    const regexManager = presets['dockerfileVersions'].regexManagers?.[0];
+
+    it(`find dependencies in file`, async () => {
+      const fileContent = `
```

Comment:
```suggestion
      const fileContent = codeBlock`
```
so you can properly indent 

https://github.com/renovatebot/renovate/blob/a48fc359b4b25eb5933a508f35603d525f41b842/lib/modules/manager/flux/extract.spec.ts#L356-L369

Reply:
Thanks, didn't know about that. Updated in https://github.com/renovatebot/renovate/pull/19966/commits/5c04bec03a8ede1ab94e501b712e74c09db05890. I've also updated `replaceString` usages to single lines to avoid wrong indentations.